packages/@react-stately/dnd/src/useDroppableCollectionState.ts

```diff
@@ -91,9 +147,9 @@ export function useDroppableCollectionState(props: DroppableCollectionStateOptio
       return false;
     },
     getDropOperation(target, types, allowedOperations) {
```

Comment:
Try to move state from stately to aria, have getDropOperation accept additional arguments here, preferably only one (try and get away with just passing isInternalDrop)